backend/app/views/spree/admin/prototypes/_prototypes.html.erb

```diff
@@ -14,7 +14,7 @@
       <tr id="row_<%= prototype.id %>" data-hook="available_row" class="<%= cycle('odd', 'even')%>">
         <td><%= prototype.name %></td>
         <td class="actions">
-          <%= link_to Spree.t(:select), select_admin_prototype_url(prototype), :class => 'ajax button select_properties_from_prototype fa fa-ok' %>
+          <%= link_to Spree.t(:select), select_admin_prototype_url(prototype), class: 'ajax button select_properties_from_prototype fa fa-ok', data: { remote: true } %>
         </td>
```

Comment:
Thanks, that's a nice improvement. I'm going to implement that in a separate PR as @Mandily has suggested a few other small improvements